automatic/ruby/README.md

```diff
@@ -2,16 +2,8 @@
 
 Ruby is a dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write.
 
-This package provides a self-contained [Windows-based installer](https://rubyinstaller.org) that includes the Ruby language, an execution environment, important documentation, and more.
-
-## Package Parameters
-
-- `/InstallDir` - Ruby installation directory, by default `c:\tools\RubyXY` where XY are major and minor version parts.
-- `/NoPath`     - Do not add ruby bin folder to machine PATH.
-
-Example: `choco install ruby --params "/NoPath /InstallDir:C:\your\install\path"`
-
 ## Notes
 
-- To install ruby development kit ruby installer provides `ridk` command. It provides easy way to install msys2 via `ridk install 1`, however, the installation is interactive. To accomplish unattended install, use [msys2](https://chocolatey.org/packages/msys2) package.
+- This package depends on the the latest major version of the ruby.install package.
+- This package do not support package parameters, please install the package [ruby.install](https://chocolatey.org/packages/ruby.install) directly when using package parameters.
```

Comment:
> This package do not support package parameters

That is not actually true, meta-packages that only differ in a `.install`, `.portable`, `.tool` and `.commandline` suffixes (bit unsure about the last two) support package parameter if the dependent package uses them.

If a package has other differences, then package parameters isn't supported (think the `python` package that depends on `python3`).

Reply:
> That is not actually true, meta-packages that only differ in a .install, .portable, .tool and .commandline suffixes (bit   unsure about the last two) support package parameter if the dependent package uses them.


I thought that it required `--applypackageparameterstodependencies`? So if you do `choco install <package> --params="/switch"`, and `<package>` depends on `<package>.install`, then the parameters do not by default get applied to `<package>install`.

https://github.com/chocolatey/choco/blob/develop/src/chocolatey/infrastructure.app/services/PowershellService.cs#L420-L425

Reply:
Not for pure metapackages (those that either differs in `.install` or `.portable`).

If it had been named differently than the install package, then yes you would have to use that argument.

Reply:
Ok, fixed. I'll need to dig into the Chocolatey source at some point to figure out where this happens.